### DISCUSSION TITLE
How to use Flyway in an Quarkus Amazon Lambda

### DISCUSSION THREAD

**Discussion by czetsuya** [quarkusio/quarkus#26252] [Q&A]
Currently, Flyway works well in a long running application using the migrate-at-start property. What's the best way to implement in a Quarkus Lambda application? I think it's not a good idea to run it with migrate-at-start set to true as it might slow the response of the service. How about having a new property migrate-at-end?

[a comment by an automated account is not shown]

**Comment by geoand** [comment]
Can you elaborate more on the `migrate-at-end` idea?

**Reply by czetsuya** [reply]
The idea is the same as migrate-at-start. But rather than doing the check at the start of the application, do it at the end or program termination. I'm talking about Lambda in particular. Doing the check at the start might add extra response time.

Or is there any other alternative/best approach to apply a Flyway script when running Quarkus on AWS Lambda?

**Reply by geoand** [reply]
But how does `migrate-at-end` actually help? That's the part I have not understood
